src/nomad_simulations/schema_packages/numerical_settings.py

```diff
@@ -123,25 +109,114 @@ class Mesh(ArchiveSection):
         shape=['n_points'],
         description="""
         The amount of times the same point reappears. A value larger than 1, typically indicates
-        a symmetry operation that was applied to the `Mesh`. This quantity is equivalent to `weights`:
+        a symmetry operation that was applied to the `Mesh`.
+        """,
+    )
 
-            multiplicities = n_points * weights
+    pruning = Quantity(
```

Comment:
So, `pruning` only makes sense given some radial grid? I didn't find any logic to raise a warning if there is none?